src/test/java/com/thealgorithms/maths/FloorTest.java

```diff
@@ -0,0 +1,28 @@
+package com.thealgorithms.maths;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+public class FloorTest {
+    @Test
+    public void testFloorWholeNumber() {
+        assertEquals(0, Floor.floor(0));
+        assertEquals(1, Floor.floor(1));
+        assertEquals(-1, Floor.floor(-1));
+        assertEquals(42, Floor.floor(42));
+        assertEquals(-42, Floor.floor(-42));
+    }
+
+    @Test
+    public void testFloorDecimalNumber() {
```

Comment:
The word `decimal` is not correct here. `Floor.floor` is defined for `double`.

Reply:
@vil02  Implemented changes as requested.